exchanges/binance/binance_wrapper.go

```diff
@@ -570,113 +570,79 @@ func (e *Exchange) UpdateOrderbook(ctx context.Context, p currency.Pair, assetTy
 	return orderbook.Get(e.Name, p, assetType)
 }
 
-// UpdateAccountInfo retrieves balances for all enabled currencies for the
-// Binance exchange
-func (e *Exchange) UpdateAccountInfo(ctx context.Context, assetType asset.Item) (account.Holdings, error) {
-	var info account.Holdings
-	var acc account.SubAccount
-	acc.AssetType = assetType
-	info.Exchange = e.Name
+// UpdateAccountBalances retrieves balances for all enabled currencies for the Binance exchange
```

Comment:
Comment is not technically true, since at no point do we check or care about the enabled currency

Reply:
So I think this is a change to all exchanges, and to make it clean we just make the comment
```
// UpdateAccountBalances retrieves currency balances
```

Fixed https://github.com/gbjk/gocryptotrader/commit/fb6f4a2d3